deploy.py

```diff
@@ -159,11 +186,21 @@ def deploy_to_chain(chain: str, contracts: List[str]):
         if legacy:
             command += " --legacy"
             print(f"Using legacy mode for {chain}")
+        
+        is_router = False
+        if short_contract_name.startswith("R"):
+            is_router = True
+            command = make_router_deployment_command(
+                command=command,
+                short_contract_name=short_contract_name,
+                chain_id=chain_id,
+                contracts_json=contracts_json,
+            )
```

Comment:
probably best that, if we have a `make_router_deployment_command`, this is consistent with batcher and peanut vault contracts (ie have a make command for all 3 of them).

Not very important though, this is just a deployment script to make it less painful to manage ops

Reply:
I thought it would be too much effort, but in fact was pretty easy to do, so I made a single command maker for all of them, good idea 👍